Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -752,6 +749,23 @@ public TriangularMovingAverage TRIMA(Symbol symbol, int period, Resolution? reso
         }
 
         /// <summary>
+        /// Creates a new T3MovingAverage indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose T3 we want</param>
+        /// <param name="period">The period over which to compute the T3</param>
+        /// <param name="volumeFactor">The volume factor to be used for the T3</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>The T3MovingAverage indicator for the requested symbol over the specified period</returns>
+        public T3MovingAverage T3(Symbol symbol, int period, int volumeFactor, Resolution? resolution = null, Func<BaseData, decimal> selector = null)
```

Comment:
My reasoning behind providing a default value in this method is because many users may not know was a `volumeFactor` is, and so providing a default allows them to only specify a period. This default value should match the value used in the `AsIndicator` extension method as well.